frontend/js/components/TableActiveRows.js

```diff
@@ -14,24 +14,22 @@ class TableActiveRows extends Component {
   }
 
   render() {
-    const { battlesResult, battleList } = this.props;
+    const { battlesResult } = this.props;
     if (!battlesResult) return <>Loading</>;
     return battlesResult.map((id) => {
-      const battle = battleList[id];
-      return <ActiveRow key={id} battle={battle} />;
+      // return <div>{id}</div>;
+      return <ActiveRow key={id} battleId={id} />;
```

Comment:
Tip: You can do:
```
return battlesResult.map((id) => <ActiveRow key={id} battleId={id} />)
```

Remember to remove the comment

Reply:
Oh nooooo sorry about that, will fix it right up